Sources/AblyAssetTrackingCore/Model/ConnectionState.swift

```diff
@@ -8,7 +8,14 @@ public enum ConnectionState: Int {
      Asset is connected to tracking system and we're receiving their position
      */
     case online
-
+    
+    /**
+     Asset state is publishing when its locations are being published but it is not able to detect subscribers or receive data from them.
+     This state allows the asset to be actively tracked, however, its features are limited compared to the online state.
+     This state can change to either online, offline or failed.
+     */
+    case publishing
```

Comment:
I was wondering why you'd added this as part of this PR — does it need to be here? I've created https://github.com/ably/ably-asset-tracking-swift/issues/625 for more broadly bringing our `TrackableState` in line with that of Android, perhaps we can do it as part of that instead?

Reply:
I've removed it as a part of a bigger merge commit: e2b1526d379b3287c70a790c4ad457b66a73fedf